addon/services/store.js

```diff
@@ -724,7 +724,12 @@ var Store = Ember.Service.extend({
     let entry, tgt;
     for ( let i = 0 ; i < ary.length ; i++ ) {
       entry = ary[i];
-      tgt = this.getById(entry.type, entry.id);
```

Comment:
If pod in project store changed, api store should notify the namespace in cluster store.
However, `this.getById('namespace', 'some-namespace-id');` will not work as it is trying to find `namespace` in the `projectStore`.